registration.js

```diff
@@ -0,0 +1,51 @@
+function autocomplete(inp, arr) {
+
+var currentFocus;
+
+inp.addEventListener("input", function(e) {
+  var countryDiv, chosenDiv, i, val = this.value;
+  closeAllLists();
+  if (!val) { return false;}
+  currentFocus = -1;
+  
+  countryDiv = document.createElement("DIV");
+  countryDiv.setAttribute("id", this.id + "autocomplete-list");
```

Comment:
do you understand what `this` keyword is stands for in here?

Reply:
yes, we are setting the "id" for the countryDiv, and this.id relates to the current created countryDiv?